package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "node-js-study",
+  "version": "1.0.0",
+  "description": "Node.js Homework Application",
+  "main": "app.js",
+  "scripts": {
+    "build": "babel ./app.js",
```

Comment:
note to correctly use imports/exports:
"babel-node": "babel-node --presets=env",
"start": "nodemon --exec npm run babel-node -- app.js"